### ISSUE TITLE
App shows empty, non-functional node after setup

### ISSUE THREAD

**Issue by wbobeirne** [ZeusLN/zeus#108]
**Describe the bug**
After entering the configuration for my node, the app shows what I would consider a "blank" node. No balance, most fields are empty or defaulted, and no actions work. I'm sure this is due to a misconfiguration, but there's no indication of that, and I can't inspect any kind of error message to figure out what might be wrong.

**Expected behavior**
Upon entering settings, a request would be made to my node's API, and any errors encountered would be presented to me so I could fix the config.

**Screenshots**

<p>
<img src="https://user-images.githubusercontent.com/649992/59370681-10f59080-8d11-11e9-81c4-8abe4124a529.png" width="300">
<img src="https://user-images.githubusercontent.com/649992/59370691-15ba4480-8d11-11e9-8a2c-4ac42f2a213e.png" width="300">
</p>

**Smartphone (please complete the following information):**
 - OS: Android 8.0.0
 - Zeus Version 0.0.9

**Comment by kaloudis** [comment]
I've seen this once before when a user passed in the RPC ports instead of REST. is this using an IPv6 host?

**Comment by wbobeirne** [comment]
Oh jeez, didn't know it used the REST API, but now I remember we'd talked about that. I'll be sure to switch it over. But I guess I'll leave the issue open and rename it, since I'm sure that's a common mistake! Might be nice to try and catch it for folks.

**Comment by kaloudis** [comment]
Yeah, I think it'd be really nice if we notified the user of this common error.

**Comment by ZeusLN** [comment]
Port field will now display as **REST** Port, and this issue has been added to the FAQ on our website but we should still detect this common error and give the user notice
